tests/test-amp-image-dimension-extractor.php

```diff
@@ -54,9 +54,9 @@ function test__normalize_url( $source_url, $expected_url ) {
 class AMP_Image_Dimension_Extractor__By_Downloading__Test extends WP_UnitTestCase {
 
 	function test__valid_image_file() {
-		$source = 'https://i0.wp.com/placehold.it/350x150.png';
+		$source = 'https://placehold.it/350x150.png';
```

Comment:
I restored the use of Photon. (I'd checked/tried many different things when investigating test failures and forgot to put this one back afterward).